### PR TITLE
Add privateIPv4CIDRs under VPC status

### PR DESCRIPTION
**Pull request by lxiaopei**
We need to expose privateIPv4CIDRs to users.

### CODE REVIEW

**Comment by dantingl** [comment]
Please remove the internal page from commit message

**Comment by lxiaopei** [comment]
> Please remove the internal page from commit message

removed, thanks.

File: Makefile [hunk 1]
```diff
@@ -40,7 +40,7 @@ help: ## Display this help.
 
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./pkg/apis/v1alpha1/;./pkg/apis/v1alpha2/" output:crd:artifacts:config=build/yaml/crd/
+	$(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1;github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha2" output:crd:artifacts:config=build/yaml/crd/
```
Comment (dantingl) [inline Makefile:43]: Is this change related?

File: pkg/apis/v1alpha1/vpc_types.go [hunk 2]
```diff
@@ -47,6 +48,8 @@ type VPCStatus struct {
 	LBSubnetPath string `json:"lbSubnetPath"`
 	// CIDR for the load balancer Subnet.
 	LBSubnetCIDR string `json:"lbSubnetCIDR"`
+        // Private CIDRs used for the VPC.
+        PrivateIPv4CIDRs []string `json:"privateIPv4CIDRs"`
```
Comment (dantingl) [inline pkg/apis/v1alpha1/vpc_types.go:52]: I remember we need to add the usage of the CIDR, not only the list

Comment (lxiaopei) [reply pkg/apis/v1alpha1/vpc_types.go:52]: as discussed, the usage of the CIDR will be added if needed in future versions.

File: Makefile (hunk 1 above)
Comment (lxiaopei) [reply Makefile:43]: if not changed, when I run "make manifests" it failed with 
/home/lxiaopei/go_projects/src/nsx-operator/bin/controller-gen rbac:roleName=manager-role crd webhook paths="./pkg/apis/v1alpha1/;./pkg/apis/v1alpha2/" output:crd:artifacts:config=build/yaml/crd/
-: main module (github.com/vmware-tanzu/nsx-operator) does not contain package github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1
-: main module (github.com/vmware-tanzu/nsx-operator) does not contain package github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha2
Error: not all generators ran successfully
run `controller-gen rbac:roleName=manager-role crd webhook paths=./pkg/apis/v1alpha1/;./pkg/apis/v1alpha2/ output:crd:artifacts:config=build/yaml/crd/ -w` to see all available markers, or `controller-gen rbac:roleName=manager-role crd webhook paths=./pkg/apis/v1alpha1/;./pkg/apis/v1alpha2/ output:crd:artifacts:config=build/yaml/crd/ -h` for usage
Makefile:43: recipe for target 'manifests' failed
make: *** [manifests] Error 1

**Review by seanpang-vmware — APPROVED** [review]
